hack/lib/serverless.bash

```diff
@@ -22,7 +42,15 @@ spec:
   name: ${OPERATOR}
   source: ${OPERATOR}
   sourceNamespace: ${OPERATORS_NAMESPACE}
+  installPlanApproval: ${INSTALL_PLAN_APPROVAL}
+  startingCSV: ${csv}
 EOF
+
+  # Approve the initial installplan automatically
+  if [ $INSTALL_PLAN_APPROVAL = "Manual" ]; then
```

Comment:
Seems like we can just ditch this if + the variable if it's constant for now anyway? Potential changes if we change to Automatic look small enough to me to just be able to change calling sites if need be.